ui/apps/platform/src/Containers/Vulnerabilities/components/FilterDropdowns.css

```diff
@@ -1,4 +1,4 @@
 /* Fixes the issue where badge counts in select components wrap to a new line */
-.workload-table-toolbar .pf-c-select__toggle-wrapper {
+.vm-filter-toolbar-dropdown .pf-c-select__toggle-wrapper {
   flex-wrap: nowrap;
 }
```

Comment:
This CSS was moved from the WorkloadCVE specific toolbar to the general component directory and applied directly to the two dropdowns that required it. This prevents the need for classes on the `<Toolbar>` components and avoids cross-importing on Workload/Node CVE pages.